flyllm/run_flyllm.py

```diff
@@ -788,6 +790,9 @@ def main(configfile, loadmodelfile=None, restartmodelfile=None):
 
 
 if __name__ == "__main__":
+    import matplotlib
+    matplotlib.use('tkagg')
```

Comment:
@kristinbranson I had to move this in order to run a test on main, do you think this will break some of your use cases?